samples/Client/DaprClient/Readme.md

```diff
@@ -2,177 +2,54 @@
-Executed State Transaction!
-State not found in store
-Retrieved state: small  yellow with etag: 1
-Saved modified state : small  orange
-Retrieved state: small  orange with etag: 2
-Saved state with old etag: 1 successfully? False
-Saved state with latest etag: 2 successfully? True
-Published deposit event!
-Done
+Running the following command will output a list of the samples included. 
+ ```sh
+ dapr run --app-id DaprClient -- dotnet run
  ```
 
-To invoke RoutingService via HTTP, run [RoutingService](../../AspNetCore/RoutingSample) or to invoke GrpcService via gRPC, run [GrpcService](../../AspNetCore/GrpcServiceSample) sample first. Then, invoke the client using the corresponding command below:-
-
-``` sh
- dapr run --app-id DaprClient -- dotnet run --useRouting true
-```
-
-or
+ Press Ctrl+C to exit, and then run the command again and provide a sample number to run the samples.
 
-``` sh
- dapr run --app-id DaprClient -- dotnet run --useGrpcsample true
-```
+  ```sh
+ dapr run --app-id DaprClient -- dotnet run 0
+ ```
 
-or to invoke RoutingService via HTTP and GrpcService via gRPC simultaneously:
-
-``` sh
- dapr run --app-id DaprClient -- dotnet run --useRouting true --useGrpcsample true
-```
+ Samples that use HTTP-based service invocation will require running the [RoutingService](../../AspNetCore/RoutingSample).
+ 
+ Samples that use gRPC-based service invocation will require running [GrpcService](../../AspNetCore/GrpcServiceSample).
 
 ## Invoking Services
+
 This solution contains a sample [RoutingSample service](../../AspNetCore/RoutingSample), which implements a simple banking application in ASP.NET core.
+
 The service provides following operations:
+
 - balance
 - withdraw
 - deposit
 
-The service is a typical HTTP service. The following sample demonstrates how to use the Dapr .NET SDK to invoke a service listening on HTTP.
-
-Operation *balance* is a GET-operation mapped as (see startup.cs in the *RoutingService*):
- ```c#
- endpoints.MapGet("{id}", Balance);
- ```
-
-It can be invoked via HTTP by using following urls.
-On Windows, Linux, MacOS:
- ```sh
-curl -X GET http://127.0.0.1:5000/17
- ```
-
-Dapr supports out-of-the-box invocation of the service operation.
-To demonstrate this, take a look at the example inside `InvokeBalanceServiceOperationAsync()`.
-
-The following example invokes another service:
-
-```c#
-
-var res = await client.InvokeMethodAsync<object>(serviceName, methodName);
-```
-
-In this example, the HttpGet method is invoked on the service with name *serviceName*. This argument specifies the name of the service, which was used when in conjunction with *dapr run –app-id serviceName*.The *route* argument specifies the route of the service endpoint.
-For example, if your service listens on HTTP, the route is typically defined by attribute *HttpPost(“route”)* or *HttpGet(“route”)*. Last argument is called metadata and it specifies GET operation in its example.
```

Comment:
It felt to me like the time was right to really dramatically simplify the content here and let the samples + API docs speak for themselves. A lot of this information reads like a tutorial - and makes sense when the richer content doesn't yet exist. However there have been several books written about this now.